android/java/org/chromium/chrome/browser/brave_news/BraveNewsAdapterFeedCard.java

```diff
@@ -0,0 +1,112 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.brave_news;
+
+import android.app.Activity;
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.chromium.brave_news.mojom.BraveNewsController;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.brave_news.BraveNewsUtils;
+import org.chromium.chrome.browser.brave_news.models.FeedItemCard;
+import org.chromium.chrome.browser.brave_news.models.FeedItemsCard;
+
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class BraveNewsAdapterFeedCard
+        extends RecyclerView.Adapter<BraveNewsAdapterFeedCard.ViewHolder> {
+    private LayoutInflater mInflater;
+    private int mType;
+    private Context mContext;
+    private Activity mActivity;
+    private View mView;
+    private LinearLayout linearLayout;
+    private LinearLayout.LayoutParams linearLayoutParams;
```

Comment:
this member `linearLayoutParams` is not used

Reply:
removed, thanks